config/vufind/EDS.ini

```diff
@@ -403,3 +403,17 @@ AU = None
 ; the LibGuides URLs if the same name is present here.
 ;url[Expensive Science Database] = https://some.url
 ;url[Cool Humanities Database] = https://another.url
+
+; Define whether to display 'Short' or 'Long' author data on the EDS search result list page.
+; - 'Short' is fetched from the RecordInfo section of a record (the 
+; 'BibRecord/BibRelationships/HasContributorRelationships/*PersonEntity/Name/NameFull' data).
+; - 'Long' is fetched from the item 'Au' group data.
+; If you do set 'ResultListPage' to 'Short' then you can define how many author names to 
+; display in AuthorNameLimit. If a record has more author names than you define in AuthorNameLimit
+; then VuFind will display the more_authors_abbrev text.
+; Also, when you set 'ResultListPage' to 'Short' you can define custom delimiters between the
+; author names in EDS.css file under: 'Author name delimiters'. 
```

Comment:
Since users may not wish to actually edit/override the full EDS.css file, maybe it would be slightly better to say "by overriding the styles defined in EDS.css" instead of just saying "in EDS.css". Or maybe that's too verbose. Feel free to ignore this suggestion if you don't like it. :-)

Reply:
thanks @demiankatz , I think that is a good addition and I have added that in my latest follow-up :)

Reply:
I would suggest to mention that the default, maybe something link "... you can change the default limiter "; " between the author names by overriding...".

Reply:
thanks @berndoberknapp I've added your suggestion in my latest followup.